src/code/z_player_lib.c

```diff
@@ -1522,6 +1522,11 @@ void Player_PostLimbDrawGameplay(PlayState* play, s32 limbIndex, Gfx** dList, Ve
             func_80090A28(this, spE4);
             func_800906D4(play, this, spE4);
         } else if ((*dList != NULL) && (this->leftHandType == PLAYER_MODELTYPE_LH_BOTTLE)) {
+            //! @bug When Player is actively using shield, the `itemAction` value will be set to -1.
+            //! If shield is used at the same time a bottle is in hand, `Player_ActionToBottle` will
+            //! return -1, which results in an out of bounds access behind the `sBottleColors` array.
+            //! A value of -1 happens to access the last three bytes of `gLinkChildBottleDL` (0x06018478)
```

Comment:
maybe clarify that 0x06018478 is the address of gLinkChildBottleDL, or write the hex value of the resulting bytes? I initially thought 0x06018478 was "the last three bytes of `gLinkChildBottleDL`" when reading this

Reply:
is this fine